JetMETCorrections/Type1MET/python/correctionTermsCaloMet_cff.py

```diff
@@ -1,13 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 ##____________________________________________________________________________||
-from JetMETCorrections.Configuration.JetCorrectors_cff import ak4CaloL2L3CorrectorChain, \
-                                                              ak4CaloL2RelativeCorrector, \
-                                                              ak4CaloL3AbsoluteCorrector, \
-                                                              ak4CaloL2L3Corrector, \
-                                                              ak4CaloL2L3ResidualCorrectorChain, \
-                                                              ak4CaloResidualCorrector, \
-                                                              ak4CaloL2L3ResidualCorrector
+from JetMETCorrections.Configuration.JetCorrectors_cff import *
```

Comment:
this will apparently import about 40 different chains instead of just 2.
Will they get purged eventually if not used or will we have all the extra objects floating in the python global space?